[2022-08-19 23:10:17,498] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: sefa_dag.get_telegram_version scheduled__2020-01-01T05:41:00+00:00 [queued]>
[2022-08-19 23:10:17,515] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: sefa_dag.get_telegram_version scheduled__2020-01-01T05:41:00+00:00 [queued]>
[2022-08-19 23:10:17,516] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-19 23:10:17,517] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-19 23:10:17,518] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-19 23:10:17,529] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): get_telegram_version> on 2020-01-01 05:41:00+00:00
[2022-08-19 23:10:17,538] {standard_task_runner.py:52} INFO - Started process 3857 to run task
[2022-08-19 23:10:17,544] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'sefa_dag', 'get_telegram_version', 'scheduled__2020-01-01T05:41:00+00:00', '--job-id', '712', '--raw', '--subdir', 'DAGS_FOLDER/sefa_dag.py', '--cfg-path', '/tmp/tmpc147qyu5', '--error-file', '/tmp/tmpkm0kjop8']
[2022-08-19 23:10:17,546] {standard_task_runner.py:80} INFO - Job 712: Subtask get_telegram_version
[2022-08-19 23:10:17,719] {task_command.py:371} INFO - Running <TaskInstance: sefa_dag.get_telegram_version scheduled__2020-01-01T05:41:00+00:00 [running]> on host fd7a784b9251
[2022-08-19 23:10:17,839] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=sefa_dag
AIRFLOW_CTX_TASK_ID=get_telegram_version
AIRFLOW_CTX_EXECUTION_DATE=2020-01-01T05:41:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-01-01T05:41:00+00:00
[2022-08-19 23:10:17,842] {python.py:173} INFO - Done. Returned value was: 1.24.0
[2022-08-19 23:10:17,891] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=sefa_dag, task_id=get_telegram_version, execution_date=20200101T054100, start_date=20220819T231017, end_date=20220819T231017
[2022-08-19 23:10:17,960] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-19 23:10:18,008] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
